demos/CellPainting_Demo.ipynb

```diff
@@ -17,7 +17,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 36,
```

Comment:
It looks like you're adding `enrichment` to the demo - that is great!

Before merging the cell painting demo, can you make sure to `restart kernel -> run all`. It is best data science practice to make sure the cells of a jupyter notebook are all executed sequentially.

Reply:
ok, will do